src/System.Slices/System/Buffers/IOutput.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Buffers
+{
+    public interface IOutput
+    {
+        Span<byte> Buffer { get; }
```

Comment:
In general we should default to Span<T> (safer) and only use Memory<T> where Span<T> is not enough.